From 39edfe5f7bf8d560c6f161c9b91f2e2e0ed9f6d7 Mon Sep 17 00:00:00 2001
From: Ricardo Salveti de Araujo <ricardo.salveti@canonical.com>
Date: Sun, 10 Apr 2011 21:57:06 -0300
Subject: [PATCH 1/3] panda: mux: Configure SYS_NIRQ2 pin in safe mode

SYS_NIRQ2 pin if not in safe mode, with audio driver enabled gates
CORE RET. Hence configured in safe mode. Audio driver is expected
to re-init the pin in the kernel.

Patch based on the SDP one from dev.omapzoom.org (59df6c45) by
Rajendra Nayak <rnayak@ti.com>

Signed-off-by: Ricardo Salveti de Araujo <ricardo.salveti@canonical.com>
---
 board/omap4430panda/omap4430panda.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/board/omap4430panda/omap4430panda.c b/board/omap4430panda/omap4430panda.c
index 962820b..9b4e457 100644
--- a/board/omap4430panda/omap4430panda.c
+++ b/board/omap4430panda/omap4430panda.c
@@ -1080,7 +1080,7 @@ static const struct omap4panda_mux omap4panda_mux[] = {
 	{ OMAP44XX_CTRL_BASE + CP(FREF_CLK2_OUT),
 					       PTD | IEN | M3  /* gpio_182 */ },
 	{ OMAP44XX_CTRL_BASE + CP(SYS_NIRQ1), PTU | IEN | M0  /* sys_nirq1 */ },
-	{ OMAP44XX_CTRL_BASE + CP(SYS_NIRQ2), PTU | IEN | M0  /* sys_nirq2 */ },
+	{ OMAP44XX_CTRL_BASE + CP(SYS_NIRQ2), M7  /* sys_nirq2 */ },
 	{ OMAP44XX_CTRL_BASE + CP(SYS_BOOT0), PTU | IEN | M3  /* gpio_184 */ },
 	{ OMAP44XX_CTRL_BASE + CP(SYS_BOOT1), M3  /* gpio_185 */ },
 	{ OMAP44XX_CTRL_BASE + CP(SYS_BOOT2), PTD | IEN | M3  /* gpio_186 */ },
-- 
1.7.4.1

